package.json

```diff
@@ -54,7 +54,22 @@
     "@babel/plugin-transform-runtime": "^7.5.5",
```

Comment:
We have a top-level psammead version and changelog. These should be updated too.